api/spec/requests/spree/api/images_controller_spec.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require 'spec_helper'
+require "private_address_check/tcpsocket_ext"
```

Comment:
Just curious why this is needed?

Reply:
This is the code that contains the error that will be raised, and that the spec tests for.

[The gem itself](jtdowney/private_address_check) seems pretty simple, and we could just copy or fork it, but it has [undergone some review](https://www.cvedetails.com/vulnerability-list/vendor_id-18563/product_id-47091/opbyp-1/Private-Address-Check-Project-Private-Address-Check.html), so I figured maybe it's worth keeping.  I'm open to a better solution, if anyone has any ideas.

Reply:
I could require it only in the sole spec where it was needed